code/src/components/Dropdown1.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './Dropdown1.css'
+
+  const Dropdown1 = ({ genre, setGenre}) => {
+
+  return (
+    
+    <form className="dropdown-genre">
+      <label className="genre-heading" htmlFor="genre" tabIndex="0">What is your favourite live music genre?</label>
+        <select
+          className="select-genre"
+          onChange={event => setGenre(event.target.value)} 
+          value={genre}
+        >
```

Comment:
I believe you should have a id in the select tag, with the same name as the htmlFor in the label.